spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -0,0 +1,281 @@
+describe('GOVUK.OptionSelect', function() {
+
+  var $optionSelectHTML, optionSelect;
+
+  beforeEach(function(){
+
+    optionSelectFixture = '<div class="govuk-option-select" tabindex="0">'+
```

Comment:
Ah cool, hadn't heard of Karma, I'll check it out. It might be worth adding this in if future work involves a lot more Javascript testing in this way, but for now I'm going to leave the tests as they are.